observability/cluster_observability_operator/cluster-observability-operator-overview.adoc

```diff
@@ -9,18 +9,21 @@ toc::[]
 :FeatureName: The Cluster Observability Operator
 include::snippets/technology-preview.adoc[leveloffset=+2]
```

Comment:
technology-preview snippet should be deleted

Reply:
The TP notices are removed in a separate PR - see https://github.com/openshift/openshift-docs/pull/85586

So, once everything is merged, this should be ok